src/careers/current-openings/_single.html

```diff
@@ -1,14 +1,13 @@
-{% extends "layout-side-nav.html" %}
+b{% extends "layout-side-nav.html" %}
 {% import "_vars-careers.html" as vars with context %}
 {%- set active_nav_id = 'current-openings' -%}
 
 {% block title -%}
-    {# TODO: Add real page meta title. #}
-    Test Opening
+    {{ career.title }}
 {%- endblock %}
 
 {% block desc -%}
-    {# TODO: Add page meta description. #}
+    {{ career.excerpt|striptags }}
```

Comment:
Mix of spaces and no spaces around `|` in this file. We haven't established a code style standard for this, but it would be good to keep it in the same format in the same file. Seems like other occurrences are with spaces.